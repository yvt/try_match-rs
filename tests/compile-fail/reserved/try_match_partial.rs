fn main() {
    // empty scrutinee is reserved for partial application
    // <https://github.com/yvt/try_match-rs/issues/3>
    let _ = try_match::try_match!(, Some(_0))(Some(42));
}
