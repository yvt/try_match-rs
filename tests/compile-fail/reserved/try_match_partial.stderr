error: partial application requires `unstable` Cargo feature
 --> tests/compile-fail/reserved/try_match_partial.rs:4:13
  |
4 |     let _ = try_match::try_match!(, Some(_0))(Some(42));
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::assert_unstable` which comes from the expansion of the macro `try_match::try_match` (in Nightly builds, run with -Z macro-backtrace for more info)
